git-link.el

```diff
@@ -176,6 +186,22 @@
 	"origin"
       remote)))
 
+(defun git-link--handler (str alist n)
+  "Return Nth element from ALIST whose `car' (a regexp) matches STR.
+
+The ALIST consists of (REGEXP FN) list elements.
+Valid ALISTs are `git-link-commit-remote-alist',`git-link-commit-alist'.
+
+Nth element of ALIST is returned at the first match of STR with REGEXP.
+
+Return nil,
+- if STR does not match with REGEXP in any of the elements of ALIST, or
+- if STR is not a string "
+  (when (stringp remote-host)
+    (nth n (cl-find-if (lambda (lst)
```

Comment:
> Though now that cl-find-if is being used is this function necessary? 

I just put it in for the `stringp` check. Also allows a good place to explain what it does in the fn's docstring.

> Also nth is not needed since we'll hold off on alist consolidation.

Would it be alright to keep that in.. just in case?